libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -957,7 +957,9 @@ bool AP_GPS::blend_health_check() const
 void AP_GPS::handle_gps_rtcm_data(const mavlink_message_t *msg)
 {
     mavlink_gps_rtcm_data_t packet;
-    mavlink_msg_gps_rtcm_data_decode(msg, &packet);
+    mavlink_msg_gps_rtcm_data_decode(msg, &packet);

```

Comment:
Unintended change? I think it's just whitespace, and there is the unneeded debug comment below that can be deleted.